plugins/unicodelib/unicodelib.cpp

```diff
@@ -1475,4 +1550,17 @@ UNICODELIB_API void UNICODELIB_CALL ulUnicodeLocaleExcludeNthWord(unsigned & tgt
         tgtLen = 0;
         tgt = 0;
     }
+}
+
+UNICODELIB_API unsigned UNICODELIB_CALL ulUnicodeLocaleFindCount(unsigned srcLen, UChar const * src, unsigned hitLen, UChar const * hit, unsigned formLen, char const * form)
+{
+    UnicodeString source(src, srcLen);
```

Comment:
If the optional parameter calls for a normalization, then they are modified by the normalization. I made the parameters in the findCount() function call constants though.